programs/git-wallet-v1/src/main.rs

```diff
@@ -0,0 +1,48 @@
+use anchor_client::{
+    anchor::{mint, transfer},
+    Client,
+};
+use solana_sdk::pubkey::Pubkey;
+use serde_json::json;
+use std::io;
+
+fn main() {
```

Comment:
I now see what you mean. This becomes a client for users to mint an NFT they can bring as ID. Maybe there exists a program for this 🤔 How do you imagine the usecase for this program?

Reply:
Yeah I was thinking this would be the monitor and it could send an instruction to the smart contract containing the pull request id and the git username. Then the contract use that to handle checking for bonk rewards, escrow receive/release, and checking merge commit for username. Working on a frame for my idea on this now. Not fully fleshed out with errors and everything but its an idea - not the way we have to go

Reply:
Yes so it would observe a PR and PR creator and relay that to the smart contract? How do you imagine the PR creator to sign the transferring of tokens to the escrow?

Reply:
I would imagine main to be a continuously running program that would poll domains fetched from our program. So it would start of every cycle by fetching `Domain`s. Then search for some syntax like `$bonk:10m` in the description of PRs, tasks or issues (bounties). It will then try to write that info to our program. If the bounties already exists then the program would reject the creation (because the seeds are unique).